docs/source/using/troubleshooting.adoc

```diff
@@ -113,6 +113,15 @@ cause passwords to fail.
 Workaround: When creating and entering passwords, wrap the string with
 single quotes in the following format: `<password>`
 
+NOTE: The characters `", $, ` and \` are still interpreted by the shell,
```

Comment:
-1 I don't think this is correct. Single quotes protects everything except other single quotes. So to use the example 'Pass\$\$d\' will literally print the backslashes and all. If you put a string into single quotes, the only thing which is problematic is the single quote itself, since it will in this case end the single quoted string. You cannot either escape the single quote within a single quoted string. You rather have to concatenate two strings. 

Escaping works in double quoted strings which is not what we describe here.

See also http://linux.101hacks.com/bash-scripting/quotes-inside-shell-script/

It might be that 'Pass\$\$d\"' actually works, but this is because of a bug. When you specify this string on the shell it will be passed as as _Pass\$\$d\"_ to Minishift. In this we pass this escaped string on to registration where we don't escape via single quotes. This means the special characters get escaped and it seemingly works. 

The statement "The characters `", $, ` and \` are still interpreted by the shell even when you wrap the string with single quotes" is, however,  wrong.

@praveen Have you tried passing a password with a space? 'hello world' for example? The proper solution is to put the password in single quotes when calling he ssh command. You also need to parse the password for potential single quotes and do something like 'my'"'"'password'